Music.py

```diff
@@ -208,6 +208,22 @@
     ("Ending Credits Part 4", 0x6A),
 )
 
+# WIP for custom ocarina songs match fanfares setting in the future currently does nothing
+ocarina_song_ids: tuple[tuple[str, int], ...] = (
```

Comment:
These are already defined in `OcarinaSongs.SONG_TABLE`. I recommend re-using that definition.

Reply:
I probably should remove this from the pr I forgot I made the changes but the table for ocarina playback and activation sequences is in a different order than in the sequence table